resources/UndocumentedAthena.md

```diff
@@ -146,7 +146,7 @@ For more sample code, please check [ddl_ctas.go](https://github.com/uber/athenad
 
 ## When the Row resultset contains Header
 
-[`GetQueryResults`](https://godoc.org/github.com/aws/aws-sdk-go/service/athena#Athena.GetQueryResults) and [`GetQueryResultsWithContext`](https://godoc.org/github.com/aws/aws-sdk-go/service/athena#Athena.GetQueryResultsWithContext) are the two functions to retrieve all Athena query results.
+[`GetQueryResults`](https://pkg.go.dev/github.com/aws/aws-sdk-go-v2/service/athena?utm_source=godoc#Client.GetQueryResults) is the function to retrieve all Athena query results.
```

Comment:
Are these lines generated somehow? Asking because of the `?utm_source=godoc`. Same question for the change below too.

Reply:
Not sure how that got in there, actually! Works without it, so I've removed them both.